clickhouse/tests/metrics.py

```diff
@@ -68,6 +68,142 @@
     'clickhouse.zk.watch',
 ]
 
+# These are the metrics that are not always available
+OPTIONAL_METRICS = [
```

Comment:
What does it depend on whether these metrics are available or not?

Reply:
The relationship between saturn and jupiter, seriously I dont know but it is a common pattern for many services that not all metrics are available on every check run. Maybe they need certain events to happen or they're emitted at a slower cadence because are expensive to compute. Services never document this.